### PR TITLE
Azure CIS 4.3.6 Ensure server parameter 'log_retention_days' is greater than 3 days for PostgreSQL Database Server

### PR DESCRIPTION
**Pull request by XOmniverse**
Azure CIS 4.3.6 Ensure server parameter 'log_retention_days' is greater than 3 days for PostgreSQL Database Server

Tested and works.

### CODE REVIEW

**Review by beagledeltasix — DISMISSED** [review]
Approved.

File: security/azure/pg_log_retention/pg_log_retention.pt [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+      field "subscriptionId" do
+        label "Subscription ID"
+      end
+      field "subscriptionName" do
+        label "Subscription Name"
+      end
+      field "id" do
+        label "Resource ID"
+      end
+      field "logRetentionDays" do
```
[the remark this answers is not in the document]
Comment (beagledeltasix) [reply security/azure/pg_log_retention/pg_log_retention.pt:182]: I guess you are doing this in the script section.

[the remark this answers is not in the document]
Comment (XOmniverse) [reply security/azure/pg_log_retention/pg_log_retention.pt:182]: Yup :)

**Review by beagledeltasix — COMMENTED** [review]
Is this the correct global readme update? Says enforce SSL not log retention?

**Review by taterdog590 — DISMISSED** [review]
approved

File: security/azure/pg_log_retention/pg_log_retention.pt [hunk 2]
```diff
@@ -0,0 +1,198 @@
+name "Azure Ensure PostgreSQL Servers Sufficient Log Retention"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report if any PostgreSQL server instances do not have log retention configured for more than 3 days. \n See the [README](https://github.com/flexera/policy_templates/tree/master/security/azure/pg_log_retention) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```
Comment (nia-vf1) [inline security/azure/pg_log_retention/pg_log_retention.pt:4]: Regarding the 2nd link - replace RightScale docs link with Flexera Automation docs link

**Review by nia-vf1 — DISMISSED** [review]

**Review by bryankaraffa — APPROVED** [review]

**Review by bryankaraffa — APPROVED** [review]

**Pull request merged**